SFR-641 item editions list

hokei opened the pull request:
[Project Reno Jira Ticket](http://jira.nypl.org/browse/RENO-XXX)

[ResearchNow Jira Ticket](http://jira.nypl.org/browse/SFR-XXX)

## **This PR does the following:**
- Adds Editions List into DS

### Front End Review:
- [ ] View [the example in Storybook](https://reno-XXX-nypl.pantheonsite.io/themes/custom/nypl_emulsify/pattern-lab/public)
- [ ] Check against the [Metronome documentation](http://themetronome.co/components/ol-24--editions-list/?fresh=true)

hokei committed:
Adding Editions List

hokei commented on src/react-components/package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nypl/design-system-react-components",
-  "version": "0.0.7",
+  "version": "0.0.10",
```
We'll need to figure out how to make new releases automatically tag and push. Backlog it goes.

helenvholmes commented on src/react-components/src/components/03-organisms/EditionsList/EditionsList.tsx:
```diff
@@ -0,0 +1,31 @@
+
+  const base_class = "edition-list";
+
+  return <ul className={bem(base_class, [], blockName)}>
```
Nit: should that array be modifiers?

helenvholmes commented on src/styles/02-molecules/edition-card/_edition-card.scss:
```diff
@@ -80,4 +80,8 @@
   }
+
+  &--list {
+    margin: 30px;
```
Instead of applying this on the list I imagine we'd want to build it into the card itself. We have helper functions for `margin-bottom` here (assuming that's the margin you want on this, which I think it is): https://github.com/NYPL/nypl-design-system/blob/development/src/styles/00-base/03-space/03-space-stack/_space-stack.scss#L21

helenvholmes commented on src/styles/CHANGELOG.md:
```diff
@@ -6,7 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
-## Prerelease
+## PRERELEASE 
+### Added 
+- - Styles for `editions-list`
```
Nit: double dash

helenvholmes reviewed (changes requested):
Some pretty small changes. Looks good once those are cleaned up.

hokei committed:
nits

hokei committed:
Publish

 - @nypl/design-system-icons@0.0.4-alpha.0
 - @nypl/design-system-react-components@0.0.11-alpha.0
 - @nypl/design-system-styles@0.0.9-alpha.0
 - @nypl/design-system-twig@0.0.7-alpha.0
 - storybook@0.0.8-alpha.0

hokei commented:
weird!  It pushes an automated publish commit to package, I wonder why it's not doing it in `development` ...

helenvholmes commented:
@hokei I'm not sure I understand what's happening here—I see you've logged a card for these so when we tackle it can we sit down and chat about it more?

hokei committed:
Modified SearchPromo story to reflect correct styles

hokei committed:
Added container component and styles cleanup

hokei committed:
tweaked breadcrumb margin override

hokei committed:
Publish

 - @nypl/design-system-react-components@0.0.11-alpha.1
 - @nypl/design-system-styles@0.0.9-alpha.1
 - storybook@0.0.8-alpha.1

helenvholmes committed:
Update ISSUE_TEMPLATE based on feedback from Reno team

helenvholmes committed:
Merge branch 'SFR-641_item_editions_list' of https://github.com/NYPL/nypl-design-system into SFR-641_item_editions_list

helenvholmes committed:
[641] Nits

helenvholmes reviewed (approved):
👍🏻

helenvholmes committed:
[641] Update changelogs
